src/components/DayPickerRangeController.jsx

```diff
@@ -107,6 +107,7 @@ export default class DayPickerRangeController extends React.Component {
     };
 
     this.isTouchDevice = isTouchDevice();
+    this.today = moment();
```

Comment:
Either that, or when different props and state will be rendered - I'm not 100% sure.